test/recipes/70-test_sslrecords.t

```diff
@@ -207,9 +208,24 @@ SKIP: {
     $fatal_alert = 0;
     $proxy->clear();
     $boundary_test_type = DATA_AFTER_KEY_UPDATE;
-    $proxy->filter(\&not_on_record_boundary);
```

Comment:
Are these ```$proxy->filter``` assignment removals just unrelated cleanup or were they functional changes?

Reply:
Unrelated clean up. The filter doesn't get cleared by a `$proxy->clear()` call so there is no need to set them every time.

Reply:
That's what it looked like, but thanks for confirming.